component/namespace-policies.jsonnet

```
@@ -58,6 +58,36 @@ local organizationNamespaces = kyverno.ClusterPolicy('organization-namespaces')
     validationFailureAction: 'enforce',
     background: false,
     rules: [
+      {
+        name: 'set-default-organization',
+        match: {
+          resources: {
+            kinds: [
+              'Namespace',
+            ],
+          },
+        },
+        context: [
+          {
+            name: 'ocpuser',
+            apiCall: {
+              urlPath: '/apis/user.openshift.io/v1/users',
+              jmesPath: "items[?metadata.name=='{{ request.userInfo.username}}']",
+            },
```

I couldn't figure out how to make the following work, any ideas:

```suggestion
            apiCall: {
              urlPath: '/apis/user.openshift.io/v1/users/{{ request.userInfo.username }}',
            },
```

---

That's weird. I thought that should work. How does it fail?

---

Resolved in the latest version: the problem was that omitting `jmesPath` led to the variable not being added to the context at all. I've now added `jmesPath='@'` which returns the current element.